### ISSUE TITLE
Permitting the combination of 011711 011722 011721 01172B

### ISSUE THREAD

**Issue by StephenMorey** [notofonts/ahom#21]
### Font: Noto Serif Ahom
NOTOSERIFAHOM-REGULAR.ttf

### Font Version
Version 2.006 ttfautohint (v 1.8.4.7-5d5b)

### Issue
Within the Tai Ahom script, there is a particular combination of the symbols 011711 011722 011721 01172B that conveys a word pronounced something like /hit/ and meaning 'do'.  At present this is realised with dotted circles. as
![image](https://github.com/user-attachments/assets/afd70938-6bfe-4961-87bd-24bff77439ef)

My understanding is that this is caused because the vowel symbol 011721 is not permitted to combine with 01172B but this should be made possible

**Comment by simoncozens** [comment]
I'm happy to do this but I would like to see an example of how the virama combines with the aa-sign visually. Simply removing the dottedCircle gives us this, which is probably not a solution:

![shape](https://github.com/user-attachments/assets/908ce4e9-0714-4e68-a429-569730f5d152)

**Comment by dscorbett** [comment]
The dotted circle is inserted by the shaping engine because USE categorizes U+1172B AHOM SIGN KILLER as VAbv whereas U+11721 AHOM VOWEL SIGN AA is VPst. The USE-compliant order <U+11711, U+11722, U+1172B, U+11721> does not get a dotted circle:
![𑜑𑜢𑜫𑜡](https://github.com/user-attachments/assets/8992c54c-599c-4035-8e24-4e5b7477bb95)
That still looks wrong, so this font still needs updating. Depending on how U+1172B is meant to interact with U+11721, USE might need updating too.

**Comment by StephenMorey** [comment]
Thank you again, Simon Cozens and DSCorbett for your responses on this. I'm have prepared the appended .pdf which contains my current best understanding of the situation with this. I'm trying to get some feedback from members of the Ahom community (via Facebook) about this. 
[The combination 011711 011722 011721 01172B.pdf](https://github.com/user-attachments/files/17386341/The.combination.011711.011722.011721.01172B.pdf)

**Comment by StephenMorey** [comment]
Hi both of you. I've had some contact with community members and I need to make a correction to what I wrote last time. 
Actually this combination representing the sound hit, and with the meaning ‘do’ consists of the following: 
AHOM LETTER HA 011711 
AHOM VOWEL SIGN I 011722 
AHOM LETTER TA 011724 (but with only the right hand descending vertical section and a join that joins it onto AHOM LETTER HA and
AHOM SIGN KILLER 01172B (virama)

The issue here is that AHOM LETTER TA is not used in it's full form but rather, for this word only, appears as just the right hand section of TA combined with HA.

It should look like this:
![image](https://github.com/user-attachments/assets/a758c694-9cb3-4cf5-bd83-de41dd0ebb7e)

but underlyingly this is the combination of 011711 011722 011704 01172B
Is this something that can be fixed at the level of the font?

**Comment by simoncozens** [comment]
Yes, sounds like it's worth making a ligature for this particular combination.

**Comment by StephenMorey** [comment]
Thanks Simon, but does that mean we need to apply to Unicode or is that something that you can do? Sorry that I'm not completely sure of the process.

**Comment by StephenMorey** [comment]
Great, thank you!

**Comment by simoncozens** [comment]
I think we want this?

<img width="218" alt="Screenshot 2024-10-17 at 10 12 11" src="https://github.com/user-attachments/assets/b096c9d0-f815-41b5-badd-02528eabcfe7">

Alternatively we could construct it using ka as the reference for the stroke and get this:

<img width="327" alt="Screenshot 2024-10-17 at 10 13 43" src="https://github.com/user-attachments/assets/82bf2019-f208-4b3f-ba9e-439f46484376">

which to my eyes looks more like the manuscripts you've shown.

But probably the first is more decipherable.

I'm pretty sure we *don't* want the right hand side of the ta, which is diagonal in this font, and so would end up like this:

<img width="226" alt="Screenshot 2024-10-17 at 10 19 19" src="https://github.com/user-attachments/assets/f86faa14-32b8-4695-bea3-241717706568">

(Or maybe we do?)

**Comment by StephenMorey** [comment]
Good morning Simon and thanks for this. I will share these with members of the Ahom community and get some response. I agree with you that the second option looks much more like what we see in the manuscripts. And that the third doesn't.

One of the issues, however, will be how it is entered in the data and how it is searched. If it is indeed stored in the data as 011711 011722 011704 01172B, it is going to be searchable which would be a benefit.

**Comment by simoncozens** [comment]
> One of the issues, however, will be how it is entered in the data and how it is searched. If it is indeed stored in the data as 011711 011722 011704 01172B, it is going to be searchable which would be a benefit.

What the font does to display it doesn't alter how it is stored in files; i.e., if people input 011711 011722 011704 01172B, that's what will be stored.

**Comment by StephenMorey** [comment]
> > One of the issues, however, will be how it is entered in the data and how it is searched. If it is indeed stored in the data as 011711 011722 011704 01172B, it is going to be searchable which would be a benefit.
> 
> What the font does to display it doesn't alter how it is stored in files; i.e., if people input 011711 011722 011704 01172B, that's what will be stored.

Thank you for that clarification. Once we have a final design, I'll talk again to people about the data entry, but at the moment I think it should still be the four symbols.

**Comment by StephenMorey** [comment]
Medini Madhab Mohan who is one of the younger generation of leaders of the Tai Ahom community and a person of considerable expertise in the manuscripts has drawn this image of what this should look like
![image](https://github.com/user-attachments/assets/0b7ce0be-a2bd-4969-9ee9-c31f3388ca44)

However, Medini will have been thinking about manuscript style and the Noto font is modern style so it might not be able to look exactly as Medini has drawn it. I'm honestly not sure how best to go about deciding this.

**Comment by StephenMorey** [comment]
Good morning. Another of the Tai Ahom experts has been consulted and come up with a different suggestion which may be more appropriate. I received this today and wanted to forward it as soon as possible
![Images of hit from Dulen](https://github.com/user-attachments/assets/f9fa546b-56a2-4363-b106-6edd9e59c7e0)

Thank you

**Comment by StephenMorey** [comment]
I should add that this is a bit closer to your first suggestion and I think somewhere between these and your first suggestion might be the most appropriate.
